Atos20/Crate
# A user will see their style at the end of the survey and can redirect…

What’s this PR do?  
- Displays a user their style based on the survey after clicking submit.
- Removes unnecessary comments.
- Fixes testing files to pass after updates to display user style.
- A user can redirect back to their subscriptions page after finishing their survey.
- Moves the survey data into its own file instead of living directly on StyleSurvey component.
- PropTypes StyleSurvey component.
- Removes any redundancies found in code.
- Clears StyleSurvey values when a user takes a survey and logs out.
 
Where should the reviewer start?  
- git pull this branch.
- web/src/modules/StyleSurvey.
- `npm start` inside web directory to launch FE and BE.
- `npm install` inside web directory if testing dependencies are not installed.
 
How should this be manually tested?  
- login with a user.
- navigate to crates.
- Ensure a user cannot take the survey if they have already taken it (check for true inside DB).
- Change flag to false and click on `+subscribe` to do a survey.
- Finish the survey and test link to subscriptions.
- Finish the survey and ensure you see your style.
- Try to take a survey again with that same user to ensure it doesn't launch twice.
- Log out with that user to ensure any data pertaining to style clears.
 
Any background context you want to provide?  
- n/a
 
What are the relevant tickets?  
- fix #51 
- fix #50 
 
Screenshots (if appropriate)  
 - n/a

Questions: 
- n/a
